nixos/tests/installer.nix

```diff
@@ -31,6 +31,9 @@ let
         # To ensure that we can rebuild the grub configuration on the nixos-rebuild
         system.extraDependencies = with pkgs; [ stdenvNoCC ];
 
+        # We need switch-to-configuration on the target
+        system.switch.enableNg = true;
```

Comment:
This should be `system.switch.enable = true;`; `enableNg` will probably go away.